settings.gradle

```diff
@@ -76,6 +76,15 @@ List projects = [
   'qa:wildfly'
 ]
 
+File examplePluginsDir = new File(rootProject.projectDir, 'plugins/examples')
```

Comment:
Why not just declare them like all the other plugins? This doesn't seem like it is better than having the plugin in `:plugins:examples:advanced-scoring`.